providers/configcat/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
     <groupId>dev.openfeature.contrib.providers</groupId>
     <artifactId>configcat</artifactId>
-    <version>0.0.2</version> <!--x-release-please-version -->
+    <version>0.1.0</version> <!--x-release-please-version -->
```

Comment:
I think this version should be updated in the README.md as well.

Reply:
At the readme the version is under[release-please](https://github.com/open-feature/java-sdk-contrib/blob/main/providers/configcat/README.md?plain=1#L7) block and should be auto update when released.